### PR TITLE
Portofolio Mobile Pull Request

### PR DESCRIPTION
**Pull request by mar4444**
This is the pull request of this project Portfolio. I have create branch with the name my-portofolio-mobile.
I created this project using html and css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="card-lang">
+            <li class="lang1">html</li>
+            <li class="lang1">css</li>
+            <li class="lang1">javaScript</li>
+          </ul>
+          <button class="card-btn">See Project</button>
+        </div>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to stick to the Figma design as close as possible. This involves font, color, padding, margin among others. Your design does not look the same as the Figma design. Kindly give a closer look to the Figma file and work your design alongside the Figma design. Take a look :point_down: 

### Your Design
your-work-design

### Figma Design
temp-work-design

File: index.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+            <div class="skill">
+             <li><h4 class="skill-title">Skills</h4></li> 
+             <li><img  src="images/Union2.svg" alt="arrow" /></li>
+            </div>
+           <hr class="new1" />
+          </div>
+        </ul>
+        </div>
+    
+    </section>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required to stick to the Figma design as close as possible. This involves font, color, padding, margin among others. Your design does not look the same as the Figma design. Kindly give a closer look to the Figma file and work your design alongside the Figma design. Take a look :point_down: 

### Your Design
your-about-section

### Figma Design

temp-about-1

temp-about-2

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @mar4444, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @mar4444, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Well-implemented suggested changes.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**